networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,68 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages `ExternalDNS` to provide name resolution for services and routes from the external DNS provider to OpenShift Container Platform.
+
+These release notes track the development of the ExternalDNS Operator in {product-title}.
+
+[id="external-dns-operator-release-notes-1.2.0"]
+== ExternalDNS Operator 1.2.0
+
+The following advisory is available for the ExternalDNS Operator version 1.2.0:
+
+* link:https://access.redhat.com/errata/RHEA-2023:7239[RHEA-2022:5867 ExternalDNS Operator 1.2 operator/operand containers]
+
+[id="external-dns-operator-1.2.0-new-features"]
+=== New features
+
+* The ExternalDNS Operator now supports AWS shared VPC. For more information, see xref:../../networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc#nw-control-dns-records-public-aws-with-VPC_creating-dns-records-on-aws[Creating DNS records on a public hosted zone for AWS by using Red Hat External DNS Operator].
```

Comment:
The link goes to a different section than the link text suggests. Based on the context, I think the link text needs to change to match the section you're linking to:

```suggestion
* The ExternalDNS Operator now supports AWS shared VPC. For more information, see xref:../../networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc#nw-control-dns-records-public-aws-with-VPC_creating-dns-records-on-aws[Creating DNS records in a different AWS Account using a shared VPC].
```